src/rlx_prv_relup.erl

```diff
@@ -145,7 +155,14 @@ make_upfrom_script(State, Release, UpFrom) ->
                             "relup successfully created!"),
             {ok, State};
         {ok,_, Module,Warnings} ->
-            ?RLX_ERROR({relup_script_generation_warn, Module, Warnings});
+            case WarningsAsErrors of
```

Comment:
FYI: I have written ticket for this issue (the error atom) to be considered at our next sprint planning in the beginning of February.